docs/core/install/linux-centos.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Install .NET Core on CentOS - .NET Core
+description: Demonstrates the various ways to install .NET Core SDK and .NET Core Runtime on CentOS.
+author: thraka
+ms.author: adegeo
+ms.date: 06/04/2020
+---
+
+# Install .NET Core SDK and .NET Core Runtime on CentOS
+
+.NET Core is supported on CentOS. This article describes how to install .NET Core on CentOS.
+
+[!INCLUDE [linux-intro-sdk-vs-runtime](includes/linux-intro-sdk-vs-runtime.md)]
+
+[!INCLUDE [linux-install-package-manager-x64-vs-arm](includes/linux-install-package-manager-x64-vs-arm.md)]
+
+## Supported distributions
+
+The following table is a list of currently supported .NET Core releases on both CentOS 7 and CentOS 8. These versions remain supported until either the version of [.NET Core reaches end-of-support](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) or the version of CentOS is no longer supported.
+
+- A ✔️ indicates that the version of CentOS or .NET Core is still supported.
+- A ❌ indicates that the version of CentOS or .NET Core isn't supported on that CentOS release.
+- When both a version of CentOS and a version of .NET Core have ✔️, that OS and .NET combination are supported.
+
+| CentOS                   | .NET Core 2.1 | .NET Core 3.1 | .NET 5 Preview |
+|--------------------------|---------------|---------------|----------------|
+| ✔️ [8](#centos-8-) | ✔️ 2.1        | ✔️ 3.1        | ✔️ 5.0 Preview |
```

Comment:
The ✔️ and ❌ do not work well with screen readers. I think they're okay occasionally, but certainly not in tables. Imagine relying on a screen reader to read a table like this, and those icons not being read - instead of being informative, the table tells them nothing. This is an accessibility concern.

Reply:
We had a discussion on this (maybe 7 months ago? I don't know if it changed? ) But the screen readers we tested did read these correctly. I personally just tried narrator and edge read aloud and both read them fine. Which is great because back then narrator didn't read them correctly and I think we sent them feedback.